libraries/libswing/src/main/java/org/pentaho/reporting/libraries/designtime/swing/ColorUtility.java

```diff
@@ -187,4 +189,17 @@ public static Color convertToDarker( final Color c ) {
     return new Color( Color.HSBtoRGB( hsb[ 0 ], hsb[ 1 ], Math.min( 1, hsb[ 2 ] / 1.2f ) ) );
   }
 
+  public static int getBrightness( Color color ) {
+    if ( color == null ) {
+      return BRIGHTNESS_THRESHOLD + 1;
+    }
+    final int r = color.getRed();
+    final int g = color.getGreen();
+    final int b = color.getBlue();
+    int brightness = ( r > g ) ? r : g;
```

Comment:
Did you write this from scratch?

Reply:
I did not! It was some that I moved around but the brightness calculation existed before